rules/Php80/Rector/Class_/ClassPropertyAssignToConstructorPromotionRector.php

```diff
@@ -142,6 +146,26 @@ public function provideMinPhpVersion(): int
         return PhpVersionFeature::PROPERTY_PROMOTION;
     }
 
+    private function processCopyDoc(Property $property, Param $param): void
+    {
+        $phpDocInfo = $property->getAttribute(AttributeKey::PHP_DOC_INFO);
+        if (! $phpDocInfo) {
+            return;
+        }
+
+        $varTag = $phpDocInfo->getVarTagValueNode();
+        if (! $varTag instanceof VarTagValueNode) {
+            return;
+        }
+
+        if ($varTag->description !== '') {
+            return;
+        }
+
+        $phpDocInfo->removeByType(VarTagValueNode::class);
+        $param->setAttribute(AttributeKey::PHP_DOC_INFO, $phpDocInfo);
+    }
```

Comment:
Could you extract this to one of existing/new doc-related services?
This should be used in any place related to docs copying.

Reply:
implemented, moved to PhpDocTypeChanger https://github.com/rectorphp/rector-src/pull/744/commits/86a459983eb7c520885ddb17a7a5f3d02e7c2d19